src/index.html

```diff
@@ -6,9 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo-container">
+        <a href="/">
+          <picture>
+            <source srcset="/images/logo.png" media="(orientation: portrait)" >
+            <img src="/images/logo.png" alt="Logo" >
+          </picture>
+        </a>
+      </div>
+      <div class="navigation-block">
+        <nav class="navigation">
+          <ul class="menu-nav">
+            <li>
```

Comment:
- Add empty lines between multiline sibling blocks of HTML
<img width="1268" alt="Screenshot 2023-10-14 at 13 37 21" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/98055117-6278-45e0-b068-e1a259c8191b">